rust/processor/src/db/common/models/emojicoin_models/models/market_24h_rolling_volume.rs

```diff
@@ -34,7 +35,8 @@ impl RecentOneMinutePeriodicStateEvent {
                         market_id: pse.market_metadata.market_id,
                         market_nonce: pse.periodic_state_metadata.emit_market_nonce,
                         transaction_version: version,
-                        period_volume: pse.volume_quote.clone(),
+                        period_quote_volume: pse.volume_quote.clone(),
```

Comment:
Will this require a cold upgrade? If so, I'd suggest using `base_` *and* `quote_` in `market_1m_periods_in_last_days`